README.md

```diff
@@ -67,7 +67,8 @@ environments.
 ## Documentation
 
 * [Setup Guide](https://firebase.google.com/docs/admin/setup/)
-* [Database Guide](https://firebase.google.com/docs/database/admin/start/)
+* [Realtime Database Guide](https://firebase.google.com/docs/database/admin/start/)
+* [Firestore Guide](https://googleapis.dev/nodejs/firestore/latest/index.html)
```

Comment:
I think this should point to https://firebase.google.com/docs/firestore

Reply:
The reason that I submitted this pr was because I was trying to get started with coding and the api for this library is different to the api used on the main docs. The PR is editable if you want to change the url.